src/main/resources/application.yaml

```diff
@@ -777,7 +777,7 @@ govnotify:
       legalRep:
         email: 66a5e934-5875-4e60-a343-356916a112c7
     bail:
-      submitAplication:
```

Comment:
please can you check its occurrence in the codebase and replace with correct spellings. One occurrence is in hearingCentrePersonalisation.